TestRunner.py

```diff
@@ -83,20 +88,54 @@ def _load_all_transformation_test_case(
         filter_test_cases = []
         package_dir = Path(__file__).resolve()  # --> TestRunner.py
         filters_dir = package_dir.parent.joinpath(search)
-        for (_, m, _) in iter_modules([filters_dir]):
-            if m in disable_tests_for:
-                continue
-            print(f"Directory = {m}")
-            t_py = import_module(f"{search}.{m}")
-            t_js = os.path.join(filters_dir, m, "test.json")
-            filter_instance = None
-            prev_class_args = {}
-            for test_case in load_test_cases(t_js):
-                class_name = test_case["class"]
-                class_args = test_case["args"] if "args" in test_case else {}
-                cls = getattr(t_py, class_name)
-
-                is_heavy = cls.is_heavy()
+
+        # Import light transformations only
+        if heavy is False and search == "transformations":
+            for m in map_transformation["light"]:
+                print(f"Directory = {m}")
+                if m in disable_tests_for:
+                    continue
+                self._load_single_transformation_test_case(m, search)
```

Comment:
We need to handle one more case where a transformation.py has multiple classes, which can have a mix of heavy and light transformation. As we are only checking the package name to distinguish b/w heavy and light, we need to pass a param 'is_heavy' in function "_load_single_transformation_test_case" and after import_module we can check for which class we need to load the test cases based on 'is_heavy' param.

Reply:
Done!